common-features/input-validation.md

```diff
@@ -748,6 +755,150 @@ The Color Palette component, while not an input, can work with validation so you
 }
 ````
 
+## Validation Mode for Simple Inputs
+
+Simple textbox-like inputs provide an option to configure the event triggering their validation. You can customize that event through the `ValidateOn` parameter they expose. It takes a member of the `ValidationEvent` enum and provides the following options:
```

Comment:
Maybe we should clarify which inputs are "simple". Is the ComboBox a simple input? What about a DateTimePicker?

```suggestion
Simple textbox-like inputs can trigger validation at different events. You can customize that through the `ValidateOn` parameter. It takes a member of the `ValidationEvent` enum and provides the following options:
```